loader_hub/zulip/base.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from typing import List, Optional
+from datetime import datetime
+import os
+import zulip
```

Comment:
could this be a lazy import within the `__init__.py` file? see other data loaders as an example

mostly because it's breaking tests atm

Reply:
@blue-note bumping this! would be good to get this fixed, then we can land

Reply:
thanks!